index.ts

```diff
@@ -61,5 +68,8 @@ export {
     // deployed addresses
     ContractAddresses,
     // types
-    RelayManagerData
+    RelayManagerData,
+    RelayData,
+    TypedRequestData,
+    RelayHubConfiguration
```

Comment:
constants and utils are used only by testing of contracts. The constants we had a conversation that since its only using the zero constant so fat we can have it locally. The utils so far is only used by the contracts on the unit testing.

Reply:
Yes, I remember, sorry 😅